packages/amplify-cli-core/src/index.ts

```diff
@@ -34,6 +34,7 @@ export type $TSContext = {
   usageData: $TSAny;
   runtime: $TSAny;
   pluginPlatform: IPluginPlatform;
+  newUserInfo?: $TSAny;
```

Comment:
Can this be more strongly typed?

Reply:
Not with ease. I don't fully know how it's being used throughout the CLI. This was added to fix TS compilation. We should refactor the functions that set this value to return the data directly instead mutating `context`, but that was out of scope for this PR.